src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { trimName } from '@/lib/utils';
+import Image from 'next/image';
+import styles from './style.module.scss';
+
+interface UserCardProps {
+  position: 1 | 2 | 3;
+  rank: string;
+  name: string;
+  points: number;
+  image: string;
+}
+const positionNames = ['first', 'second', 'third'];
+
+const TopUserCard = ({ position, rank, name, points, image }: UserCardProps) => {
```

Comment:
By convention, let's make the component inside the file match the file name. So here, we should name the component TopThreeCard instead of TopUserCard. I also think it makes it more clear what the component is too 👍